### PR TITLE
Mdd m borg/combine unload distance

### PR DESCRIPTION
**Pull request by MddMBorg**
For issue #7250 added some code to get the length of the trailer and add this to the alignLength offset for combine self unload (was sometimes colliding header with trailer while turning when the trailer was long)

### CODE REVIEW

File: CombineAIDriver.lua [hunk 1]
```diff
@@ -1341,8 +1341,15 @@ function CombineAIDriver:startSelfUnload()
 		self.courseAfterPathfinding = nil
 		self.waypointIxAfterPathfinding = nil
 		local targetNode = fillRootNode or bestTrailer.rootNode
+		local trailerRootNode = bestTrailer.rootNode
+		local trailerLength = bestTrailer.sizeLength
+		self:debug('Trailer Length : %d', trailerLength)
+
+		local dX, dY, dZ = localToLocal(trailerRootNode, targetNode, 0, 0, 0)
```
Comment (pvaiko) [inline CombineAIDriver.lua:1348]: you can just do local _, _, dz = ... if you don't need dx and dy.

**Review by pvaiko — CHANGES_REQUESTED** [review]
Thanks for the pull request, just one minor thing.

File: CombineAIDriver.lua (hunk 1 above)
Comment (MddMBorg) [reply CombineAIDriver.lua]: Done in latest commit

File: CombineAIDriver.lua [hunk 2]
```diff
@@ -1341,8 +1341,15 @@ function CombineAIDriver:startSelfUnload()
...
+		self:debug('ax = %d, ay = %d, az = %d', dX, dY, dZ)
```
[the remark this answers is not in the document]
Comment (MddMBorg) [reply CombineAIDriver.lua]: Removed in latest commit

**Pull request merged**